Adding gridpack cards for AMSB charginos Run3 production

borzari opened the pull request:
Adding gridpack cards for production of chargino+chargino and chargino+neutralino in the AMSB process using MG5, where the mass difference between chargino and neutralino is small enough to make the chargino decay inside the tracker volume (long-lived particle for EXO disappearing tracks Run 3 analysis). Merging process with extra jet using the MLM jet matching scheme, where xqcut was determined with MG5 locally and qcut is set as 1.5*xqcut. Cards are for several mass/lifetime of charginos. The parameters in the customize card were extracted from the same processes being executed in MG5 locally, using the parameters in the SLHA files from the [Run 2 analysis](https://github.com/cms-sw/genproductions/tree/master/genfragments/ThirteenTeV/DisappTrksAMSB) events generation.

menglu21 commented:
Hi @borzari you submitted several hundreds of cards, the difference of those cards are the mass point, right? if so, please only create template cards for one mass point and use script to make cards for others.

borzari commented:
Hi @menglu21. Thank you for your comment. Indeed there are some similarities between the customize files. I created templates for the process and run cards that can be shared in between every mass point/lifetime, but with the customize cards, I need at least one for each mass point. In this new commit I use a script to generate the folders containing the gridpack cards. I still didn't remove the old cards, but will do another commit to remove them if you think the last changes are ok.

borzari commented:
Hi @menglu21. Do you have any other comments/suggestions about this PR?

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/AMSB_chargino/10000cm/AMSB_chargino_M1000GeV_ctau10000cm/AMSB_chargino_M1000GeV_ctau10000cm_run_card.dat:
```diff
@@ -0,0 +1,144 @@
+# WARNING: Do not use for interference type of computation           *
+#*********************************************************************
+   True  = use_syst      ! Enable systematics studies
+#
+systematics = systematics_program ! none, systematics [python], SysCalc [depreceted, C++]
+['--mur=0.5,1,2', '--muf=0.5,1,2', '--pdf=errorset', '--alps=0.5,1,2'] = systematics_arguments ! see: https://cp3.irmp.ucl.ac.be/projects/madgraph/wiki/Systematics#Systematicspythonmodule
+# Syscalc is deprecated but to see the associate options type'update syscalc'
```
those lines can be removed (also for other cards)

menglu21 reviewed (commented):
Hi, sorry for the late reply and thanks for the update, I just have some minor comments

borzari commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/AMSB_chargino/10000cm/AMSB_chargino_M1000GeV_ctau10000cm/AMSB_chargino_M1000GeV_ctau10000cm_run_card.dat:
```diff
@@ -0,0 +1,144 @@
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     'lhapdf'    = pdlabel     ! PDF set
+     325500    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
In here, should I remove the four lines that you mentioned, or should I do like this:

     'lhapdf'    = pdlabel     ! PDF set
     $DEFAULT_PDF_SETS = lhaid
     $DEFAULT_PDF_MEMBERS = reweight_PDF

borzari commented:
Hi @menglu21. Thank you for the comments. I have answered them and now I have a question: should I also upload my hadronizer config files? If yes, which folder should I put them? Would it be better to create a new PR for this?

menglu21 commented:
> Hi @menglu21. Thank you for the comments. I have answered them and now I have a question: should I also upload my hadronizer config files? If yes, which folder should I put them in? Would it be better to create a new PR for this?

Hi, it's not required to upload hadronizer here but of course you can. yes, you can create a new PR for the hadronizer. btw, since you already have script to get all the mass point, could you remove the cards for the mass points, and only keep the template cards and script, i.e, keep only necessary files here.

borzari commented:
> In here, should I remove the four lines that you mentioned, or should I do like this:
> 
> ```
>  'lhapdf'    = pdlabel     ! PDF set
>  $DEFAULT_PDF_SETS = lhaid
>  $DEFAULT_PDF_MEMBERS = reweight_PDF
> ```

Hi @menglu21. Just removed the files and added some changes to the run_card template. Could you please answer the question above? I am going to create a new PR for the hadronizers. In which directory should I add those?

menglu21 commented:
> > In here, should I remove the four lines that you mentioned, or should I do like this:
> > ```
> >  'lhapdf'    = pdlabel     ! PDF set
> >  $DEFAULT_PDF_SETS = lhaid
> >  $DEFAULT_PDF_MEMBERS = reweight_PDF
> > ```
> 
> Hi @menglu21. Just removed the files and added some changes to the run_card template. Could you please answer the question above? I am going to create a new PR for the hadronizers. In which directory should I add those?

sorry for overlooking it. the following is preferred.

'lhapdf'    = pdlabel     ! PDF set
 $DEFAULT_PDF_SETS = lhaid
 $DEFAULT_PDF_MEMBERS = reweight_PDF

borzari commented:
Hi @menglu21. Just added the changes to the run cards. Thank you for pointing that out! I also added a `README.md` with the simple instruction to create the cards and how they are going to be displayed in the folder.

I am going to make another PR for the hadronizers. In which folder should I add those?

menglu21 commented:
> Hi @menglu21. Just added the changes to the run cards. Thank you for pointing that out! I also added a `README.md` with the simple instruction to create the cards and how they are going to be displayed in the folder.
> 
> I am going to make another PR for the hadronizers. In which folder should I add those?

you can update them in https://github.com/cms-sw/genproductions/tree/master/genfragments/ThirteenPointSixTeV
